rest-api/test/unit_test/api_test/physical_measurements_api_test.py

```diff
@@ -55,15 +55,15 @@ def test_insert(self):
 
     em1 = Measurement(measurementId=pm_id * 1000,
                       physicalMeasurementsId=pm_id,
-                      codeSystem="http://loinc.org",
-                      codeValue="55284-4",
+                      codeSystem="http://terminology.pmi-ops.org/CodeSystem/physical-measurements",
+                      codeValue="systolic-diastolic-blood-pressure-1",
                       measurementTime=now,
                       bodySiteCodeSystem="http://snomed.info/sct",
                       bodySiteCodeValue="368209003")
     bp1 = Measurement(measurementId=pm_id * 1000 + 1,
                       physicalMeasurementsId=pm_id,
-                      codeSystem="http://loinc.org",
-                      codeValue="8480-6",
+                      codeSystem="http://terminology.pmi-ops.org/CodeSystem/physical-measurements",
+                      codeValue="systolic-blood-pressure-1",
```

Comment:
Do we not currently have a test that parses PMs from JSON? It would be good to show the "pick preferred" logic in action.

Reply:
Yes, we do -- this test in particular, does. See measurements-as-fhir.json above.

Reply:
Yes, so we do. (Sorry, that was... pretty clear from the change history.)